src/components/apps/launch/ResourceRequirements.js

```diff
@@ -116,52 +116,84 @@ const StepResourceRequirementsForm = ({
+                            <MenuItem key={index} value={size}>
+                                {size}
+                            </MenuItem>
+                        ))}
+                    </FastField>
+                </Grid>
+                <Grid item xs={12}>
+                    <FastField
+                        id={buildID(baseId, ids.RESOURCE_REQUESTS.TOOL_MEM)}
+                        name={`requirements.${index}.min_memory_limit`}
+                        label={t("minMemory")}
+                        component={FormSelectField}
+                        renderValue={formatGBValue}
+                    >
+                        {minMemoryList.map((size, index) => (
+                            <MenuItem key={index} value={size}>
+                                {formatGBListItem(size)}
+                            </MenuItem>
+                        ))}
+                    </FastField>
+                </Grid>
+                <Grid item xs={12}>
+                    <FastField
+                        id={buildID(
+                            baseId,
+                            ids.RESOURCE_REQUESTS.MIN_DISK_SPACE
+                        )}
+                        name={`requirements.${index}.min_disk_space`}
+                        label={t("minDiskSpace")}
+                        component={FormSelectField}
+                        renderValue={formatGBValue}
+                    >
+                        {minDiskSpaceList.map((size, index) => (
+                            <MenuItem key={index} value={size}>
+                                {formatGBListItem(size)}
+                            </MenuItem>
+                        ))}
+                    </FastField>
+                </Grid>
             </Grid>
-            <Grid item xs={12}>
-                <FastField
-                    id={buildID(baseId, ids.RESOURCE_REQUESTS.MIN_DISK_SPACE)}
-                    name={`requirements.${index}.min_disk_space`}
-                    label={t("minDiskSpace")}
-                    component={FormSelectField}
-                    renderValue={formatGBValue}
-                >
-                    {minDiskSpaceList.map((size, index) => (
-                        <MenuItem key={index} value={size}>
-                            {formatGBListItem(size)}
-                        </MenuItem>
-                    ))}
-                </FastField>
+            <br />
+            <Grid container spacing={1}>
+                <Grid item xs={12}>
+                    <Typography variant="subtitle2">
+                        {t("selectMaxs")}
+                    </Typography>
+                </Grid>
+                <Grid item xs={12}>
+                    <FastField
+                        id={buildID(baseId, "MAX_CPU")}
+                        name={`requirements.${index}.max_cpu_cores`}
+                        label={t("maxCPUCores")}
+                        component={FormSelectField}
```

Comment:
Should we add some `helperText` to this field explaining to the user that the max setting here is what will count as their actual usage for this analysis?

Reply:
Alternatively, since we currently have a large block of text above the minimum select lists explaining what effect these settings have on submission wait times, then we could have another block of text above the max settings explaining the effect of the max CPU setting.

Reply:
@psarando  - Do you have any suggestions for the text ? I turning up blank 😞

Reply:
We should get input from @johnworth and 411?